routes/auth.routes.js

```diff
@@ -0,0 +1,81 @@
+const bcryptjs = require('bcryptjs')
+const { Router } = require('express')
+const app = require('../app')
+const User = require("../models/User.model")
+ const mongoose = require('mongoose')
+const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard")
+
+
+const saltRounds = 10
+const router = new Router()
+
+//Rutas
+router.get('/signup', isLoggedIn,  (req,res,next) => res.render('auth/signup'))
```

Comment:
Debería ser `isLoggedOut`,` isLoggedIn` te redirecciona automaticamente al log in.